
SR: main_with_error_in_import.py
    ST: Collect tasks (line: 0)
        L: E: "Error when importing module 'main_with_error_in_import' at location <path>." (line: 100)
        STB: ModuleNotFoundError: No module named 'module_that_does_not_exist'
    ET: ERROR
ER: ERROR